src/styles/_catalog.scss

```diff
@@ -1,22 +1,33 @@
+$cardWidth: 200px;
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
```

Comment:
```suggestion
  display: grid;
  justify-content: center;
```
catalog not centered
![Screenshot from 2023-03-25 17-54-53](https://user-images.githubusercontent.com/33029864/227728135-f72c8684-dc77-4a58-bea1-ed728a61c269.png)